src/components/DisplayNames/index.native.js

```diff
@@ -4,7 +4,7 @@ import Text from '../Text';
 
 // As we don't have to show tooltips of the Native platform so we simply render the full display names list.
 const DisplayNames = props => (
-    <Text style={props.textStyles} numberOfLines={props.numberOfLines}>
+    <Text testID={props.testID} style={props.textStyles} numberOfLines={props.numberOfLines}>
```

Comment:
This highlights something that I think might already be on the "iffy" side. My concern is having a bunch of testIDs everywhere in the code that end up making a mess eventually.

In my tests with the LHN, I ended up using `accessabilityHint` as a way to query for some of the components I was looking for, and I felt like that it was at least making a small contribution toward accessibility.

Do you have any initial thoughts on how we can try to get a handle on this problem early (querying the rendered tree for specific components)?

Reply:
Managed to replace 99% of these.

Reply:
Cool, thanks! I'll give it a look.